<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>620</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>FIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>345</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="VitesseTimerlSlider">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>400</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="PourcentagelSlider">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>290</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>260</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>   Pourcentage Sable</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>380</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>  Vitesse Eau</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PourcentagelineEdit">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>330</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>530</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcul</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Perco">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>610</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>480</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>570</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>430</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>NouveauDes</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>820</x>
     <y>669</y>
    </hint>
    <hint type="destinationlabel">
     <x>677</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VitesseTimerlSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>VitessevalueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>804</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>774</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PourcentagelSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>PourcentagevalueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>858</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>811</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>StartCalcul()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>853</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>702</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ArretCalcul()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>872</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>StartResultat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>875</x>
     <y>616</y>
    </hint>
    <hint type="destinationlabel">
     <x>643</x>
     <y>593</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>NouveauDessin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>907</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Demarrage()</slot>
  <slot>VitessevalueChanged()</slot>
  <slot>PourcentagevalueChanged()</slot>
  <slot>StartCalcul()</slot>
  <slot>ArretCalcul()</slot>
  <slot>StartResultat()</slot>
  <slot>NouveauDessin()</slot>
 </slots>
</ui>
